iroha/src/account.rs

```diff
@@ -15,6 +15,8 @@ pub struct Account {
     pub id: Id,
     /// Asset's in this `Account`.
     pub assets: BTreeMap<<Asset as Identifiable>::Id, Asset>,
+    /// Associated instruction.
+    pub instruction: Option<Box<Instruction>>,
```

Comment:
You need to put instructions into `assets` as an Asset with `instruction@global` Asset Definition.